### PR TITLE
ADEV-57 Missing titles in doc pages

### PR DESCRIPTION
**Pull request by spg**

### CODE REVIEW

File: site/src/main/markdown/gwtp/advancedfeatures/Custom-Proxy-Class.md [hunk 1]
```diff
@@ -21,20 +19,20 @@ To do a standard proxy...
 The following example shows you how to build a basic custom proxy that is also a place. This example uses code splitting, if you don't want this replace `AsyncProvider` with `Provider` and `CodeSplitProvider` with `StandardProvider`.
 
 ```
-public class MyPresenterProxyPlaceImpl extends ProxyPlace<MyPresenter> 
+public class MyPresenterProxyPlaceImpl extends ProxyPlace<MyPresenter>
```
Comment (Pacane) [inline site/src/main/markdown/gwtp/advancedfeatures/Custom-Proxy-Class.md]: Can we specify the language for syntax highlight?

File: site/src/main/markdown/gwtp/advancedfeatures/Merge-Locale.md [hunk 2]
```diff
@@ -19,17 +19,17 @@ The exact process is as follows:
...
-generated in the folder of the `UiBinder` file, so your resources are 
-spread across your directories. mergelocale looks for all of these and 
-merge them intelligently in a central location: com.google.gwt.i18n.client. This is legit since this package is the 
-directory of the `LocalizableResource`, a superinterface common to both 
+GWT looks for the `LocalizableResource???.properties` files in the
+directory of your class implementing Constants or Messages, and in
+the directory of any super interfaces. The problem with `UiBinder`
+translations is that the associated `LocalizableResource` files are
+generated in the folder of the `UiBinder` file, so your resources are
+spread across your directories. mergelocale looks for all of these and
```
Comment (Pacane) [inline site/src/main/markdown/gwtp/advancedfeatures/Merge-Locale.md]: `mergelocale`

File: site/src/main/markdown/gwtp/advancedfeatures/Merge-Locale.md [hunk 3]
```diff
@@ -19,17 +19,17 @@ The exact process is as follows:
...
+merge them intelligently in a central location: com.google.gwt.i18n.client. This is legit since this package is the
```
Comment (Pacane) [inline site/src/main/markdown/gwtp/advancedfeatures/Merge-Locale.md]: merges

File: site/src/main/markdown/gwtp/advancedfeatures/Merge-Locale.md [hunk 4]
```diff
@@ -19,17 +19,17 @@ The exact process is as follows:
...
+directory of the `LocalizableResource`, a superinterface common to both
 Constants and Messages.
 
-When compiling a localized GWT project, one has to 
- * compile 1st time, to generate extras locales 
- * run mergelocale and update translations 
- * compile 2nd time, so translations are included in the results
\ No newline at end of file
+When compiling a localized GWT project, one has to
+ * compile 1st time, to generate extras locales
```
Comment (Pacane) [inline site/src/main/markdown/gwtp/advancedfeatures/Merge-Locale.md]: extra locales

File: site/src/main/markdown/gwtp/advancedfeatures/Merge-Locale.md [hunk 5]
```diff
@@ -19,17 +19,17 @@ The exact process is as follows:
...
+ * run mergelocale and update translations
```
Comment (Pacane) [inline site/src/main/markdown/gwtp/advancedfeatures/Merge-Locale.md]: `mergelocale`

File: site/src/main/markdown/gwtp/advancedfeatures/Custom-Proxy-Class.md (hunk 1 above)
Comment (spg) [inline site/src/main/markdown/gwtp/advancedfeatures/Custom-Proxy-Class.md]: Nope :(

File: site/src/main/markdown/gwtp/resources/Migrating-from-07-to-10.md [hunk 6]
```diff
@@ -1,3 +1,5 @@
+#Migrating from v. 0.7 to 1.0
```
Comment (Pacane) [inline site/src/main/markdown/gwtp/resources/Migrating-from-07-to-10.md]: missing space

File: site/src/main/markdown/gwtp/resources/Migrating-from-11-to-12.md [hunk 7]
```diff
@@ -1,3 +1,5 @@
+#Migrating from v. 1.1 to 1.2
```
Comment (Pacane) [inline site/src/main/markdown/gwtp/resources/Migrating-from-11-to-12.md]: Missing space

File: site/src/main/markdown/gwtp/resources/index.md [hunk 8]
```diff
@@ -1,3 +1,5 @@
+#Maven Configuration
```
Comment (Pacane) [inline site/src/main/markdown/gwtp/resources/index.md]: missing space

File: site/src/main/markdown/gwtp/sampletutorial/index.md [hunk 9]
```diff
@@ -1,14 +1,12 @@
-#Simple Nested Sample
+#Sample tutorial
```
Comment (Pacane) [inline site/src/main/markdown/gwtp/sampletutorial/index.md]: missing space

File: site/src/main/markdown/gwtp/sampletutorial/index.md [hunk 10]
```diff
@@ -1,14 +1,12 @@
...
+The goal of this tutorial is to help you get a complete overview using GWTP's Model-View-Presenter architecture. Part one will focus mainly on nested presenters. For a complete example on how to use GWTP in Ecplise, take a look at the GettingStarted tutorial.
```
Comment (Pacane) [inline site/src/main/markdown/gwtp/sampletutorial/index.md]: Eclipse

**Pull request merged**